.github/workflows/workflow.yml

```diff
@@ -219,34 +219,6 @@ jobs:
       run: |
         tox -e py${{ matrix.python-version }} -- -m 'not integration and not unstable'
 
-  platform_checks_sync_aea_loop:
```

Comment:
Do not remove; comment out! We need to re-enable them later.

Reply:
done

Reply:
see below re reintroducing this with the stub

Reply:
done